src/renderer/services/auto-updater.service.ts

```diff
@@ -35,11 +56,59 @@ export class AutoUpdaterService {
         });
 
         this.progressService.show(this.downloadProgress$, true);
-        
         return promise;
     }
 
     public quitAndInstall() {
         this.ipcService.sendLazy("install-update");
     }
+
+    public getLastAppVersion(): string {
+        return this.configurationService.get("last-app-version");
+    }
+
+    public setLastAppVersion(){
```

Comment:
It will be better if you update the last version by the version passed in params